apps/archive_broadcast/broadcast.py

```diff
@@ -32,7 +32,7 @@
 logger = logging.getLogger(__name__)
 # field to be copied from item to broadcast item
 FIELDS_TO_COPY = ['urgency', 'priority', 'anpa_category', 'type',
-                  'subject', 'dateline', 'slugline', 'place']
+                  'subject', 'cvs', 'dateline', 'slugline', 'place']
```

Comment:
You might have to do the same thing in `archive_rewrite.py`.

May be move `common` fields for `take, broadcast and update` can be moved common.py?